membership-attribute-service/app/services/IdentityAuthService.scala

```diff
@@ -22,7 +22,8 @@ class IdentityAuthService(apiConfig: IdapiConfig)(implicit ec: ExecutionContext)
       .map(user => Option(user))
       .handleError { err =>
         if(err.isInstanceOf[UserCredentialsMissingError])
-          SafeLogger.error(scrub"invalid request as no token or cookie provided - unable to authorize user - $err", err)
+          SafeLogger.warn(s"invalid request as no token or cookie provided - unable to authorize user - $err." +
+            s"Request attempted: ${requestHeader.method} ${requestHeader.path} with headers ${requestHeader.headers}", err)
```

Comment:
have you tested locally to see how this is formatted? I only ask because it might be worth considering a mkString('\n') or some other tweaking to make it more readable

Reply:
Good idea. Adding in some newlines the logging would now look like: 
![image](https://user-images.githubusercontent.com/3072877/63429510-29211480-c412-11e9-841c-f36487cf0cba.png)